python/huggingfaceserver/huggingfaceserver/model.py

```diff
@@ -191,6 +238,84 @@ def load(self) -> bool:
             self.tokenizer.add_special_tokens({"pad_token": "[PAD]"})
         logger.info(f"successfully loaded tokenizer for task: {self.task}")
 
+        if self.use_triton:
```

Comment:
We can consider adding this option to `PredictorConfig`, so any KServe model is able to launch the triton server in the `load` when the option is enabled.  
https://github.com/kserve/kserve/blob/master/python/kserve/kserve/model.py#L89